src/test/java/edu/ucsb/cs156/courses/jobs/UpdateCourseDataJobTests.java

```diff
@@ -382,9 +361,9 @@ void test_if_stale_and_is_not_stale() throws Exception {
 
     // Assert
 
-    String expected = "Data is not stale for [MATH 20211]";
+    /*String expected = "Data is not stale for [MATH 20211]";
 
     assertEquals(expected, jobStarted.getLog());
-    verify(isStaleService, times(1)).isStale(eq("MATH"), eq("20211"));
+    verify(isStaleService, times(1)).isStale(eq("MATH"), eq("20211"));*/
```

Comment:
I think you need to remove the commented code based on the professor's requested change

Reply:
Absolutely correct.  This must be removed before this is merged for points.

Reply:
I fixed it !